documentation/common/proc-download-procedure.adoc

```diff
@@ -14,7 +14,7 @@ ifdef::Downloading[]
 endif::Downloading[]
 
 ifndef::Downloading[]
-* Download and extract the `{ZipInstallFile}` file from the {ZipDownload}.
+* Download and extract the `{ZipInstallFile}` file from the {ZipDownload}. 
```

Comment:
So for this release there will be two bundles, one suitable for < Kube 1.16 (and OpenShift 3.11) and another suitable for newer Kubernetes and all OpenShift 4 versions.  How best shall this be presented?   Feel free to commit to this PR or I'll make a suggestion tomorrow.  @hmaclean73